sim/globals.ts

```diff
@@ -1370,3 +1370,48 @@ interface PokemonModData {
 	originalSpecies?: string; // Mix and Mega
 	[key: string]: any;
 }
+
+interface Group {
+	symbol: string,
+	id: string,
+	name: string,
+
+	root?: boolean,
+	inherit?: string,
+	jurisdiction?: string,
+	roomonly?: boolean,
+	globalonly?: boolean,
+	console?: boolean,
+	lockdown?: boolean,
+	hotpatch?: boolean,
+	ignorelimits?: boolean,
+	promote?: string,
+	makeroom?: boolean,
+	editroom?: boolean,
+	editprivacy?: boolean,
+	ban?: boolean,
+	mute?: boolean,
+	lock?: boolean,
+	receivemutedpms?: boolean,
+	forcerename?: boolean,
+	ip?: boolean,
+	alts?: boolean,
+	modlog?: boolean,
+	broadcast?: boolean,
+	declare?: boolean,
+	announce?: boolean,
+	modchat?: boolean,
+	modchatall?: boolean,
+	potd?: boolean,
+	forcewin?: boolean,
+	battlemessage?: boolean,
+	tournaments?: boolean,
+	gamemoderation?: boolean,
+	gamemanagement?: boolean,
+	minigame?: boolean,
+	game?: boolean,
+
+	// Needed for room<rank> settings and to allow for custom settings on other
+	// servers.
+	[key: string]: any
```

Comment:
`[key: string]: unknown`?

`[key: string]: boolean | string`?

Reply:
I didn't want to make it `boolean | string` in case other servers use settings that aren't those types.

I'm way out of date with TypeScript, I didn't know `unknown` existed. That'd probably fit better